src/browser/modules/Frame/FrameTitlebar.tsx

```diff
@@ -127,138 +96,19 @@ type FrameTitleBarProps = FrameTitleBarBaseProps & {
 }
 
 function FrameTitlebar(props: FrameTitleBarProps) {
-  const [editorValue, setEditorValue] = useState(props.frame.cmd)
-  const [renderEditor, setRenderEditor] = useState(props.frame.isRerun)
+  const frame = props.frame
```

Comment:
Any reason not to use destructuring here? ( `const { frame } = props` )

Reply:
no